core-web/libs/block-editor/src/lib/extensions/bubble-link-form/plugins/bubble-link-form.plugin.ts

```diff
@@ -242,7 +246,10 @@ export class BubbleLinkFormView {
         const { href: link = '', target } = this.editor.isActive('link')
             ? this.editor.getAttributes('link')
             : this.editor.getAttributes(ImageNode.name);
-        const blank = target ? target === '_blank' : true;
+        const blank = target ? target === '_blank' : false;
```

Comment:
`const blank = target === '_blank'` isn't this the same?

Reply:
We can even default it to `false` or something else in the line 246 if it's possible